src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -255,7 +255,7 @@ private void OnProjectChanged(object sender, IProjectContextInfo project)
             }
             else
             {
-                EmitRefreshEvent(timeSpan, RefreshOperationSource.ProjectsChanged, RefreshOperationStatus.NoOp, isUIFiltering: false, duration: 0);
+                EmitRefreshEvent(timeSpan, RefreshOperationSource.ProjectsChanged, RefreshOperationStatus.NoOp, isUIFiltering: false);
```

Comment:
If it's not 0 then what being emitted now?

Reply:
It seems like nothing gets logged.

The description says: 

> Removes zero-durations in PM UI refresh events that resulted in NoOp to avoid skewing data analysis.


How where these messing with the data analysis? Can't the data analysis exclude noop?

Reply:
If you see 0 in an NoOp without knowing how the event is emitted, it may make you think that NoOp operation took zero-time to execute. If it's an artificial zero, I think it's better to remove it.